packages/runtime/src/public/createApp.ts

```diff
@@ -0,0 +1,93 @@
+// Note: We should be careful of what we import in this module, we don't want to load the entire runtime here
+// until createApp is called, since we set some globals here, which will be read by other runtime modules
+import '@mobilestack-xyz/runtime/src/missingGlobals'
+import Config from 'react-native-config'
+import DeviceInfo from 'react-native-device-info'
+import { ToggleableOnboardingFeatures } from 'src/onboarding/types'
+
+// This will evolve. We should be mindful of breaking changes.
+// This structure should scale well as we add more features
+// and makes it clear what's core configuration vs optional features.
+export interface PublicAppConfig {
+  registryName: string
+  displayName: string
+  deepLinkUrlScheme: string
+  iosAppStoreId?: string
+
+  // Optional features/capabilities
+  features?: {
+    firebase?: boolean
+    sentry?: {
+      clientUrl: string
+    }
+    // TODO: what's the marketing name for this?
+    cloudBackup?: {
+      auth0Domain: string
+      auth0ClientId: string
+      web3AuthClientId: string
+    }
```

Comment:
As an example, Cloud Backup is disabled by default if not set.
If someone wants to enable it, they need to pass all required properties.

Reply:
But maybe we'll enable by default 😄